pom.xml

```diff
@@ -69,12 +69,42 @@
 			<email>guenther.niess@web.de</email>
 			<timezone>+1</timezone>
 		</contributor>
+    	<contributor>
+			<name>Konstantin Yakimov</name>
+			<roles>
+				<role>Developer</role>
+			</roles>
+			<email>kiakimov@gmail.com</email>
+			<timezone>+3</timezone>
+		</contributor>
 	</contributors>
-	<scm>
-		<connection>scm:svn:http://svn.igniterealtime.org/svn/repos/tinder/trunk/</connection>
-		<developerConnection>scm:svn:https://svn.igniterealtime.org/svn/repos/tinder/trunk/</developerConnection>
-		<url>http://fisheye.igniterealtime.org/viewrep/tinder/trunk/</url>
-	</scm>
+
+    <scm>
+		<connection>scm:git:https://github.com/igniterealtime/tinder.git</connection>
+		<developerConnection>scm:git:git@github.com:kiakimov/tinder.git</developerConnection>
+		<url>https://github.com/igniterealtime/tinder</url>
+	    <tag>HEAD</tag>
+    </scm>
+
+    <repositories>
+        <repository>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+            <id>jcenter</id>
+            <name>jcenter</name>
+            <url>http://jcenter.bintray.com</url>
+        </repository>
+    </repositories>
+    
+    <distributionManagement>
+        <repository>
+            <id>bintray</id>
+            <name>bintray</name>
+            <url>https://api.bintray.com/maven/kyakimov/maven/tinder</url>
```

Comment:
This should be maven central where tinder is distributed: http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22tinder%22

Although I think we need to change the groupid from 'org.igniterealtime' to 'org.igniterealtime.tinder' for the next tinder release on OSS sonatype.